### PR TITLE
expression: implement vectorized evaluation for `builtinUncompressSig`

### PR DESCRIPTION
**Pull request by shihongzhi**
### What problem does this PR solve? 

implement vectorized evaluation for builtinUncompressSig , for #12106

### What is changed and how it works?

	$go test -v -benchmem -bench=BenchmarkVectorizedBuiltinEncryptionFunc -run=BenchmarkVectorizedBuiltinEncryptionFunc -args "builtinUncompressSig"
	goos: darwin
	goarch: amd64
	pkg: github.com/pingcap/tidb/expression
	BenchmarkVectorizedBuiltinEncryptionFunc/builtinUncompressSig-VecBuiltinFunc-4         	    4460	    275791 ns/op	  156016 B/op	    3292 allocs/op
	BenchmarkVectorizedBuiltinEncryptionFunc/builtinUncompressSig-NonVecBuiltinFunc-4      	    4332	    366714 ns/op	  156016 B/op	    3292 allocs/op
	PASS
	ok  	github.com/pingcap/tidb/expression	3.640s

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by qw4990** [comment]
@shihongzhi Friendly ping.

**Comment by shihongzhi** [comment]
> Please fix the CI problem:
> 
> ```
> [2019-10-15T15:24:19.830Z] FAIL: builtin_encryption_vec_test.go:43: testEvaluatorSuite.TestVectorizedBuiltinEncryptionFunc
> [2019-10-15T15:24:19.830Z] 
> [2019-10-15T15:24:19.830Z] builtin_encryption_vec_test.go:44:
> [2019-10-15T15:24:19.830Z]     testVectorizedBuiltinFunc(c, vecBuiltinEncryptionCases)
> [2019-10-15T15:24:19.830Z] bench_test.go:848:
> [2019-10-15T15:24:19.830Z]     // do not forget to call ResizeXXX/ReserveXXX
> [2019-10-15T15:24:19.830Z]     c.Assert(getColumnLen(output, testCase.retEvalType), Equals, input.NumRows())
> [2019-10-15T15:24:19.830Z] ... obtained int = 1855
> [2019-10-15T15:24:19.830Z] ... expected int = 1024
> ```

Fixed the CI problem

File: expression/builtin_encryption_vec.go [hunk 1]
```diff
@@ -330,11 +331,59 @@ func (b *builtinSHA1Sig) vecEvalString(input *chunk.Chunk, result *chunk.Column)
...
+	for i := 0; i < n; i++ {
+		if buf.IsNull(i) {
+			result.AppendNull()
+			continue
+		}
+
+		payload := buf.GetString(i)
+
+		if len(payload) == 0 {
+			result.AppendNull()
```
Comment (Reminiscent) [inline expression/builtin_encryption_vec.go]: Maybe here we should do ```result.Append("")``` ? What's your opinion? @qw4990  @shihongzhi

Comment (Reminiscent) [inline expression/builtin_encryption_vec.go]: Maybe here we should do ```result.AppendString("")``` ? What's your opinion? @qw4990  @shihongzhi

Comment (qw4990) [reply expression/builtin_encryption_vec.go]: Yes, you are right.
Please fix this problem @shihongzhi .

**Pull request merged**